app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptorAdapter.java

```diff
@@ -36,4 +64,132 @@ private EncryptorAdapter(final IServiceLocator serviceLocator) {
         return new ModuleWrapper<>(new EncryptorAdapter(serviceLocator));
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String hashWithoutSalt(final String pass) throws EncryptionException {
```

Comment:
Good use case, forgot about that 😅 In that case, can you rename the method either to `hashPassword` or to "`hashString` and change the parameter name to `string`"